src/main.py

```diff
@@ -7,102 +9,57 @@
-    edges.append(Edge(nodes[13], nodes[14], curved=True, center=np.array([-6,-6]), clockwise=True)) # right from north
-    edges.append(Edge(nodes[13], nodes[10], curved=True, center=np.array([ 6,-6]), clockwise=False)) # left from north
-
-    edges.append(Edge(nodes[15], nodes[10])) # straight from west
-    edges.append(Edge(nodes[15], nodes[ 8], curved=True, center=np.array([-6, 6]), clockwise=True)) # right from west
-    edges.append(Edge(nodes[15], nodes[12], curved=True, center=np.array([-6,-6]), clockwise=False)) # left from west
-
-    # routes
-    routes.append(Route([edges[ 1], edges[ 8], edges[ 4]])) # straight from south
-    routes.append(Route([edges[ 1], edges[ 9], edges[ 2]])) # right from south
-    routes.append(Route([edges[ 1], edges[10], edges[ 6]])) # left from south
-
-    routes.append(Route([edges[ 3], edges[11], edges[ 6]])) # straight from east
-    routes.append(Route([edges[ 3], edges[12], edges[ 4]])) # right from east
-    routes.append(Route([edges[ 3], edges[13], edges[ 0]])) # left from east
-
-    routes.append(Route([edges[ 5], edges[14], edges[ 0]])) # straight from north
-    routes.append(Route([edges[ 5], edges[15], edges[ 6]])) # right from north
-    routes.append(Route([edges[ 5], edges[16], edges[ 2]])) # left from north
-
-    routes.append(Route([edges[ 7], edges[17], edges[ 2]])) # straight from west
-    routes.append(Route([edges[ 7], edges[18], edges[ 0]])) # right from west
-    routes.append(Route([edges[ 7], edges[19], edges[ 4]])) # left from west
-
-    # vehicles.append(Vehicle(0,routes[0],0,8,0,2.23,4.90,1.25,'assets/sedan.png')) # south to north
-    # vehicles.append(Vehicle(1,routes[1],0,8,0,2.23,4.90,1.25,'assets/sedan.png')) # south to east
-    vehicles.append(Vehicle(2,routes[2],0,8,0,2.23,4.90,1.25,'assets/sedan.png')) # south to west
-
-    # vehicles.append(Vehicle(3,routes[3],0,8,0,2.23,4.90,1.25,'assets/sedan.png')) # east to west
-    # vehicles.append(Vehicle(4,routes[4],0,8,0,2.23,4.90,1.25,'assets/sedan.png')) # east to north
-    # vehicles.append(Vehicle(5,routes[5],0,8,0,2.23,4.90,1.25,'assets/sedan.png')) # east to south
-
-    vehicles.append(Vehicle(6,routes[6],0,8,0,2.23,4.90,1.25,'assets/sedan.png')) # north to south
-    # vehicles.append(Vehicle(7,routes[7],0,8,0,2.23,4.90,1.25,'assets/sedan.png')) # north to east
-    # vehicles.append(Vehicle(8,routes[8],0,8,0,2.23,4.90,1.25,'assets/sedan.png')) # north to west
-
-    vehicles.append(Vehicle(9,routes[9],0,8,0,2.23,4.90,1.25,'assets/sedan.png')) # west to east
-    # vehicles.append(Vehicle(10,routes[10],0,8,0,2.23,4.90,1.25,'assets/sedan.png')) # west to south
-    # vehicles.append(Vehicle(11,routes[11],0,8,0,2.23,4.90,1.25,'assets/sedan.png')) # west to north
-
-    manager = Manager(np.array([0,0]), 50)
-
-    # scenery
-    # 2 rects for road,
-
+    
+    preset_name = sys.argv[1]
+    manager = load_preset("presets/" + preset_name, nodes, edges, routes, vehicles)
```

Comment:
let's make the path to the preset be absolute, and not strictly within presets. (User has to put python3 src/main.py presets/PRESET.json)

I think this is better because a user might have a preset downloaded or edited somewhere else